### PR TITLE
phinetravel

### PR DESCRIPTION
**Pull request by Josephineoderland**
## Netlify link
https://phinetravel.netlify.app/

### CODE REVIEW

File: src/App.jsx
Comment (AntonellaMorittu) [inline src/App.jsx:1]: Very good components structure in the App.jsx file!

File: src/Components/Input/Dropdown/dropdown.css
Comment (AntonellaMorittu) [inline src/Components/Input/Dropdown/dropdown.css:1]: Gret job grouping component +. stylesheet inside a folder 👍  don't forget to format the css files! 🪄 Also the name should be Uppercase  for all stylesheets

**Review by AntonellaMorittu — APPROVED** [review]
Hi Josephine, congratulations delivering a very well made app! I appreciate your attention to detail and the effort to customise the radio buttons! The app satisfies the requirements, please format the CSS files, I left a comment in the code. Well done with responsiveness and accessibility, you did a great job this week! 👏

File: src/Components/Form/Form.jsx [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+              />
+            ) : (
+              <Text
+                name="reasonPur"
+                label="Reason for selecting 'Something else':"
+                placeholder="write your purpose:"
+                onChange={handleChange}
+                value={formState.reasonPur}
+                error={formState.errors.reasonPur}
+              />
```
Comment (viktoria-olo) [inline src/Components/Form/Form.jsx:188]: This was a really cool idea that you implemented really well!

File: src/Components/Input/Radio/radio.css [hunk 2]
```diff
@@ -0,0 +1,113 @@
+.hidden-radio {
```
Comment (viktoria-olo) [inline src/Components/Input/Radio/radio.css:1]: Very impressed you got the custom radio buttons to work 😁

**Review by viktoria-olo — COMMENTED** [review]
It looks awesome, both the code and the final outcome! Also well done on the accessibility aspect. Great job!

**Pull request closed without merging**